.eslintrc

```diff
@@ -13,7 +13,7 @@
   ],
   "rules": {
     "@typescript-eslint/no-var-requires": 0,
-    "no-console": 1,
+    "no-console": 0,
```

Comment:
Please revert this and put the ignoring rule in the file where console.log using

Reply:
Done, [link to commit](https://github.com/iden3/js-merkletree/commit/c9dfa45fde67810ec81c97c42db7e4b15cf3ebb7)